src/Bridges/intervalbridge.jl

```diff
@@ -48,6 +48,19 @@ function MOI.get(model::MOI.ModelLike, a::MOI.ConstraintDual, c::SplitIntervalBr
     return lower_dual > -upper_dual ? lower_dual : upper_dual
 end
 
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintBasisStatus, c::SplitIntervalBridge)
+    lower_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.lower)
+    if lower_stat == MOI.NONBASIC || lower_stat == MOI.NONBASIC_AT_LOWER
```

Comment:
`lower` is a `GreaterThan` constraint so should never return `NONBASIC_AT_LOWER`. It helps to check for this value but we should print a warning if we see it. Similar comment for `upper`.